apps/fabric-website/scripts/createFlightConfig.js

```diff
@@ -31,6 +31,31 @@ module.exports = function createFlightConfigTaskFactory() {
   };
 };
 
+module.exports.createPublicFlightConfigTask = function() {
```

Comment:
Would you mind adding info about what the difference is between the two configs, and what this part of the change is meant to enable?

Reply:
yes!